lib/govuk_sidekiq/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukSidekiq
-  VERSION = "0.0.1"
+  VERSION = "0.0.2"
```

Comment:
It's probably OK in this case but we usually bump versions after merge in case someone else's code goes in first.